libs/constants/src/consts/jwt.const.ts

```diff
@@ -0,0 +1,3 @@
+export const WEB_JWT_EXPIRY_TIME = '15m';
+export const GAME_JWT_EXPIRY_TIME = '24h';
+export const REFRESH_JWT_EXPIRY_TIME = '5d';
```

Comment:
I'd argue these should be namespaced, e.g. `JWT_WEB_EXPIRY_TIME` and `JWT_GAME_EXPIRY_TIME` etc.